package.exs

```diff
@@ -3,15 +3,15 @@ defmodule Erlzk.MixFile do
 
   def project do
     [app: :erlzk,
-     version: "0.6.5",
+     version: "0.6.5+klarna1",
```

Comment:
Non-semver version will cause trouble for hex package users.

Delete this file if we are not going to publish it to hex.

if we are, then:
we can switch to use rebar3 for hex package management then there is no dependency to elixir.
we do not have permission to publish to erlzk repo in hex.
so we'll need to use another name like the other two listed here:
https://hex.pm/packages?_utf8=%E2%9C%93&search=erlzk&sort=recent_downloads

Reply:
Thanks, I wasn't aware! I think I'll just drop this file, I have no intention in publishing this to hex.